### DISCUSSION TITLE
I have an error while insttalling aihwkit

### DISCUSSION THREAD

**Discussion by patrick-chen-98** [IBM/aihwkit#520] [Installation Q&A]
I posted this issue https://github.com/IBM/aihwkit/issues/505#issue-1706344167
now, I have cuda 11.8 installed, I tried again, it gave the following error, I have pytorch 2.0.1 and python 3.10
<img width="842" alt="image" src="https://github.com/IBM/aihwkit/assets/79097916/b95a2ef3-d2fb-4502-8216-2af5259f80a0">

**Comment by maljoras** [comment]
Hi @patrick-chen-98, 
Can you post a little more of the error output ?  This does not tell anything. Do you actually have a GPU in your system?

**Comment by patrick-chen-98** [comment]
> Hi @patrick-chen-98, Can you post a little more of the error output ? This does not tell anything. Do you actually have a GPU in your system?

Thank you for replying. I have an RTX4080 and an A2000 on my two PC. I will post more error output soon

**Comment by imanehmz** [comment]
Hi @patrick-chen-98 , I had similar issues too and I failed to install it on windows, I used these exact steps on the notebook and it worked for me on Google Colab with GPU.
[aihwkit_installation_notebook.zip](https://github.com/IBM/aihwkit/files/12017937/aihwkit_installation_notebook.zip)
